spec/services/people/importer_spec.rb

```diff
@@ -0,0 +1,111 @@
+RSpec.describe People::Importer do
+  subject(:service) { described_class.new(nomis_attributes) }
+
+  let(:nomis_attributes) do
+    {
+      prison_number: 'G8872UH',
+      latest_booking_id: 1_093_139,
+      last_name: 'JAVAS',
+      first_name: "EF'LIAICO",
+      middle_names: 'MATTHEWS BARNARDOS',
+      date_of_birth: '1989-10-01',
+      aliases: nil,
+      pnc_number: '05/886838E',
+      cro_number: '610592/05U',
+      gender: gender.nomis_code,
```

Comment:
could `gender` and `ethnicity` be hard-coded values too, to catch any future accidental changes?

Reply:
you may need to change `let(:gender) {...}` and `let(:ethnicity) {...}`  statements to `let!`